src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,56 @@
+package view;
+
+import domain.Ladder;
+import domain.Players;
+import java.util.List;
+
+public class OutputView {
+
+  private final int MAKE_BLANK_FIVE = 6;
+
+  public void showUpperPhase(Players players, Ladder ladder) {
+    System.out.println("실행결과\n");
+    showPlayers(players);
+    showLadder(ladder);
+  }
+
+  public void showPlayers(Players players) {
+    String result = "";
+    for (int i = 0; i < players.numberOfPlayers(); i++) {
+      String playersName = players.getPlayerByIndex(i).getPlayerNameString();
+      String blank = makingBlank(playersName.length());
+      result += playersName + blank;
+    }
+    System.out.println(result);
+  }
+
+  public void showLadder(Ladder ladder) {
+    for (int i = 0; i < ladder.getDepthValue(); i++) {
+      List<Boolean> lineValue = ladder.getLineValue(i);
+      String ladderParts = "|";
+      String result = drawLadder(lineValue, ladderParts);
+      System.out.println(result);
+    }
+  }
+
+  private String drawLadder(List<Boolean> lineValue, String ladderParts) {
+    for (int j = 0; j < lineValue.size(); j++) {
+      if (lineValue.get(j)) {
+        ladderParts += "-----|";
+      }
+      if (!lineValue.get(j)) {
+        ladderParts += "     |";
+      }
```

Comment:
다음과 같이 작성해볼 수도 있겠네요 :)
```suggestion
      if (lineValue.get(j)) {
        ladderParts += "-----|";
        continue;
      }
      ladderParts += "     |";
```

Reply:
라인 수를 줄일 수 있겠네요! 라인 수의 제한이 있을 때라면 더 효과적일 것 같아서 수정했습니다.